bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v18/V18PsbtRpc.scala

```diff
@@ -0,0 +1,34 @@
+package org.bitcoins.rpc.client.v18
+
+import org.bitcoins.rpc.client.common.Client
+import org.bitcoins.rpc.jsonmodels.{
+  AnalyzePsbtResult,
+  JoinPsbtsResult,
+  UtxoUpdatePsbtResult
+}
+import org.bitcoins.rpc.serializers.JsonSerializers._
+import play.api.libs.json._
+
+import scala.concurrent.Future
+
+trait V18PsbtRpc {
+  self: Client =>
+
+  /**
+    * Analyzes and provides information about the current status of a PSBT and its inputs
+    * @see https://bitcoincore.org/en/doc/0.18.0/rpc/rawtransactions/analyzepsbt/
+    */
+
+  def analyzePsbt(psbt: String): Future[AnalyzePsbtResult] = {
+    bitcoindCall[AnalyzePsbtResult]("analyzepsbt", List(JsString(psbt)))
+  }
+
+  def joinPsbts(txs: Seq[String]): Future[JoinPsbtsResult] = {

```

Comment:
The return type here would be better as `Future[String]` (and `Future[Psbt]`, once that's implemented (not on this PR!))